MovingBlocks/Terasology
# Remove hackish and not multiplayer ready crouch visuals.

### Contains

This removes the constant camera up/down movement in multiplayer when you crouch.

It also makes it so that while "default croaching" is enabled with key "x", presseing the temporary croach button "ctrl" will make the character temporarly stop crouching. This is analog to how it is for running.

This change removes all the hackish existing player height and eye height modificaitons during crouching. The old code had the following issues:
- the client ordered the server to change the eye height based on possibly outdated eye height values it had received previously from the server
- character height and eye height change got triggered by client instead of being a result of movement mode change
- client triggered redundant movement mode changes that would also
  automatically result from movement input
- movementDebugCommands were used to do eye height and height modification. The commands are hackish too and not really something that should be used in non debug code.
- the player height change triggered a teleportation of the player which messes up further with the movement prediction system


### How to test

Start a headless server and join as client. Then try crouching. Movement speed should be slightly slower while crouching.
